package.json

```diff
@@ -27,20 +27,21 @@
     "lint:js": "eslint --ext .js --ignore-path .gitignore migrations/**/*.js",
     "lint:sol": "solium -d contracts/",
     "lint:dydx": "python util/lintcontracts.py",
-    "coverage": "./node_modules/.bin/solidity-coverage",
+    "coverage": "node scripts/coverage.js",
     "node": "ganache-cli",
     "debug": "truffle debug",
     "build:js": "rm -rf dist/ && tsc && cp -r build/wrappers dist/build",
+    "build:cov": "npm run copy_tew && npm run build:wrappers && npm run clean_contract_json",
     "docker_node": "ganache-cli -i 1313 -d -p 8545 -h 0.0.0.0 --db=/home/.ganache",
     "clean_contract_json": "ts-node ./scripts/CleanBuild.ts",
     "deploy": "truffle migrate --network=$NETWORK --reset && npm run save_deployed_addresses",
     "deploy_kovan": "NETWORK=kovan npm run deploy",
     "deploy_mainnet": "NETWORK=mainnet npm run deploy",
     "deploy_test": "npm run reset_test_evm && npm run migrate -- --network test --reset && npm run snapshot_test_evm && npm run clean_contract_json",
-    "deploy_coverage": "RPC_NODE_URI=http://0.0.0.0:8555 npm run reset_test_evm && npm run migrate -- --network coverage --reset && RPC_NODE_URI=http://0.0.0.0:8555 npm run snapshot_test_evm && npm run clean_contract_json",
+    "deploy_coverage": "RPC_NODE_URI=http://127.0.0.1:8555 npm run reset_test_evm && COVERAGE=true npm run migrate -- --network coverage && RPC_NODE_URI=http://127.0.0.1:8555 npm run snapshot_test_evm && npm run clean_contract_json",
     "deploy_test_ci": "RPC_NODE_URI=http://0.0.0.0:8545 npm run reset_test_evm && npm run migrate -- --network test_ci --reset && RPC_NODE_URI=http://0.0.0.0:8545 npm run snapshot_test_evm && npm run clean_contract_json",
     "test_only": "NODE_ENV=test jest --setupFiles dotenv/config --runInBand",
-    "test_cov": "npm run deploy_coverage && NETWORK_ID=1002 COVERAGE=true RPC_NODE_URI=http://0.0.0.0:8555 npm run test_only -- --forceExit",
+    "test_cov": "npm run deploy_coverage && NETWORK_ID=1002 COVERAGE=true RPC_NODE_URI=http://127.0.0.1:8555 npm run test_only -- --forceExit --verbose",
     "ci_test": "NODE_ENV=test RPC_NODE_URI=http://0.0.0.0:8545 jest --setupFiles dotenv/config --runInBand",
```

Comment:
Main changes here are:
+ Separating the compile and build steps for coverage (because in 0.7.x instrumented contracts reside in a special folder in the root directory - in lieu of copying entire project into a temp folder)
+ Using 127.0.0.1 (instead of docker url?) 
+ Added --verbose to jest  (not necessary - did this for debugging)